src/inmanta/config.py

```diff
@@ -114,7 +118,12 @@ def get(
 
         opt = cls.validate_option_request(section, name, default_value)
 
-        val = cfg.get(section, name, fallback=default_value)
+        val = os.environ.get(f"INMANTA_{section.upper()}_{_to_env_name(name)}", default=None)
```

Comment:
I would put all the logic to calculate the full name of the environment variable together in one method. Currently this logic is spread over the get() method and the _to_env_name() method, which makes the code hard to read.

Reply:
Changed function to _get_env_var()